movie_api/asgi.py

```diff
@@ -0,0 +1,16 @@
+"""
+ASGI config for move_api project.
```

Comment:
There is a typo in the project name. It should be 'movie_api' instead of 'move_api'.